pkgs/by-name/ca/cargo-shear/package.nix

```diff
@@ -13,12 +21,19 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "sha256-DQjLW6NtUPERT82zIlrGRk3FffU/EfLBMgAeFfFdDxE=";
 
-  meta = with lib; {
+  # https://github.com/Boshen/cargo-shear/blob/a0535415a3ea94c86642f39f343f91af5cdc3829/src/lib.rs#L20-L23
+  SHEAR_VERSION = version;
+  passthru.tests.version = testers.testVersion {
+    package = cargo-shear;
+    command = "cargo-shear --version";
```

Comment:
```suggestion
```
this is the default https://github.com/NixOS/nixpkgs/blob/860e65d27036476edfb85dd847d982277880b143/pkgs/build-support/testers/default.nix#L60